velox/connectors/hive/TableHandle.h

```diff
@@ -28,7 +28,7 @@ using SubfieldFilters =
 
 class HiveColumnHandle : public ColumnHandle {
  public:
-  enum class ColumnType { kPartitionKey, kRegular, kSynthesized };
+  enum class ColumnType { kPartitionKey, kRegular, kSynthesized, kRowIndex };
```

Comment:
Can you add the new comment here?

Reply:
done

Reply:
Some more feedbacks from teammate, can we reformat the comment this way (also using `///`)?
```c++
 enum class ColumnType {
    kPartitionKey, 
    kRegular, 
    kSynthesized, 

    /// ...
    kRowIndex };
```

Reply:
update @Yuhta thank you !